contents/teams/surveys/objectives.mdx

```diff
@@ -1,37 +1,62 @@
 
-- Do 5 customer interviews
-- Find the top 5 reference customers and build a relationship with them
-- Do a case study with a top reference customer
+- ✅ iOS
+- ✅ React Native
+- 🛠️ Android
+- 🛠️ Flutter
 
-# Goal 2: Improve quality
+# 📈 Goal 2: Level up survey analysis
 
-There are a couple of reported bugs that were not addressed quickly, we'd like to speed up this process and fix them.
+You asked — we listened. We're making survey analysis more useful and enjoyable, starting with a brand new global analytics page.
 
-- Reduce [SLA](https://en.wikipedia.org/wiki/Service-level_agreement) to fix bugs from months to weeks
-- Reduce [paper cut](https://en.wikipedia.org/wiki/Lingchi) issues by 50%
-    - Currently around ~30 [open bugs](https://github.com/PostHog/posthog/issues?q=is%3Aopen+is%3Aissue+label%3Afeature%2Fsurveys+label%3Abug)
+- [Global analytics view](https://github.com/PostHog/posthog/issues/28418)
+- A cleaner, more useful survey responses page
+- Extra polish and UX improvements across the board
 
-# Goal 3: Finish up [WIP](https://en.wikipedia.org/wiki/Work_in_process) features
+# 🎨 Goal 3: Survey customization
 
-We've started and nearly finished a couple of features our customers requested, but they are not [GA](https://en.wikipedia.org/wiki/Software_release_life_cycle#General_availability_(GA)) yet, the goal is to finish them up, gather feedback and put them in front of customers.
+A lot of people use the API to tweak their surveys. Let’s bring those use cases into the product.
 
-- [Adaptive Sampling](https://github.com/PostHog/posthog/issues/25722)
-    - Currently under a feature flag
-- [Partial Responses](https://github.com/PostHog/posthog/issues/19300)
-    - There are Draft PRs
-- [Survey Actions](https://github.com/PostHog/posthog/issues/27378)
-    - Currently under a feature flag
-- [AI Summaries](https://github.com/PostHog/posthog/issues/27379)
-    - Currently under early access features
+- Better wait period controls
+- [Alignment customization](https://github.com/PostHog/posthog/issues/28479)
+- [Inline surveys](https://github.com/PostHog/posthog/issues/30097)
+- A handful of other power-user options
 
-# Goal 4: Mobile Surveys
+# 💬 Goal 4: Make Key Customers Super Happy
 
-We want to bring the same functionality to Mobile that we have for Web Surveys, [GH Issue](https://github.com/PostHog/posthog/issues/25085).
+We’ve had some great convos with customers — now let’s turn those into lasting partnerships.
 
-- React Native
-- iOS
-- Android
-- Flutter
+- Build a case study with a power user
+- Be lightning-fast at resolving their issues
+
+# 🧰 Goal 5: Templates, Templates, Templates
+
+Right now, duplicating and customizing surveys across projects/environments isn’t great. Let’s fix that.
+
+- [Save custom templates](https://github.com/PostHog/posthog/issues/27309)
```

Comment:
i'd remove this as already feels stretched

Reply:
this is part of the key customers ask